backend/src/routes/api/owners.ts

```diff
@@ -0,0 +1,128 @@
+import express, { Request, Response } from "express";
+import Owner from "../../models/owner";
+
+const router = express.Router();
+
+//@route    Post api/owners
+//@desc     Register new owner
+//@access   Public
+router.post("/", async (req: Request, res: Response) => {
+  const { name, lastname, address, phone, dni, cuit, email } = req.body;
+
+  //Simple validation
+  if (!fieldsAreValid(req.body)) {
+    return res.status(400).json({ msg: "Please enter all fields" });
+  }
+
+  //Check for existing owner
+  const existingOwner = await Owner.findOne({ cuit: cuit });
```

Comment:
Un pequeño truquillo, si la clave y el valor se llaman igual, podes directamente pasarlo así:
```suggestion
  const existingOwner = await Owner.findOne({ cuit });
```